docs/TiDBAPI

```diff
@@ -0,0 +1,50 @@
+#  TiDB HTTP API
+
+#### 
+> 1. Get present TiDB ' s status, about connections, version and git_hash
+
+```
+curl http://{TiDBIP}:10080/status
```

Comment:
10080 is the default port, and it can be changed in the config when start. I think we'd better tells that this is the default port or just use STATUS_PORT?